Formula/apache-arrow.rb

```diff
@@ -4,6 +4,7 @@ class ApacheArrow < Formula
   url "https://www.apache.org/dyn/closer.cgi?path=arrow/arrow-0.15.1/apache-arrow-0.15.1.tar.gz"
   sha256 "9a2c58c72310eafebb4997244cbeeb8c26696320d0ae3eb3e8512f75ef856fc9"
   head "https://github.com/apache/arrow.git"
+  revision 1
```

Comment:
```
  * C: 7: col 3: `revision` (line 7) should be put before `head` (line 6)
```

Please squash that change into the original commit

Reply:
Done.